roles/satellite-clone/tasks/backup_satellite_version_check.yml

```diff
@@ -0,0 +1,37 @@
+- name: check for satellite answers file
+  command: tar zxf {{ backup_dir }}/config_files.tar.gz  etc/foreman-installer/scenarios.d/satellite-answers.yaml --to-stdout
+  register: satellite_answers
+  ignore_errors: true
+  no_log: true
+
+- name: setting fact - satellite_answers
+  set_fact:
+    clone_satellite_answers: "{{ (satellite_answers.rc | int) == 0 }}"
+    cacheable: true
+
+- name: set satellite_version to 6.2 if detected
```

Comment:
to match the name of the next task perhaps name this `set satellite_version to 6.2 if satellite_answers.yaml exists`

Reply:
Thanks, since it will be updated to 6.3 if its a 6.3 backup, how about we say `Temporarily set satellite_version to 6.2 if satellite_answers.yaml exists`